src/chain-wallet-libs/bindings/wallet-js/js/package.json

```diff
@@ -1,11 +1,14 @@
 {
-  "name": "js",
-  "version": "0.0.1",
+  "name": "wallet-js-api",
```

Comment:
Can we consider using `wallet-js` as the package name? It looks like the suffix `-api` doesn't add value.

Reply:
we have the same name for the wasm package, should we rename it for `wallet-wasm-js` ?

Reply:
We probably also can consider removing `-js` because both packages are JS packages: `wallet` and `wallet-wasm`